samples/requirements.txt

```diff
@@ -1,4 +1,5 @@
 --extra-index-url https://download.pytorch.org/whl/cpu
+openvino
```

Comment:
why not `openvino-genai` ?

Reply:
`openvino-genai` isn't required for conversion.

Reply:
I suppose this [requirements.txt](https://github.com/openvinotoolkit/openvino.genai/pull/1048/files/f1d320f83b28511345e0c46ddcbe61f13ce413a2#diff-e1d35130d879e8dab202c2e78ae00e2197e6586726b92fe2f9086d5bc528fd68) is for both conversion and run

E.g. whisper sample relies on `librosa` which is mentioned in this requirements file.

should we have 2 requirements - for deployment and development ?

Reply:
I was hoping to avoid having two `requirements.txt` because the deployment one would just contain `openvino-genai`. But I see now that we could have a general `deployment-requirements.txt` for consistency. Will add and rename `requirements.txt`->`export-requirements.txt`.

Reply:
maybe we can also have requirements.txt which includes both of them? for simplicity and for CI

Inside each requirements we can have comments which describe the purpose of the file (to avoid describing in each sample readme)